### PR TITLE
Tiling

### PR DESCRIPTION
**Pull request by EduardoLR10**
Normal setup for test, just try it out.

### CODE REVIEW

**Comment by mtrsk** [comment]
@EduardoLR10 since we are addding a bunch of pngs to the repo, can you do:
```
git-lfs track *.png
git-lfs track *.obj
```

File: server/database/migrations/000003_map.sql [hunk 1]
```diff
@@ -1,12 +1,22 @@
 CREATE TYPE map.TILE_TYPE AS ENUM(
+       'EMPTY',
```
Comment (z-silver) [inline server/database/migrations/000003_map.sql:2]: Does it even make sense for a tile to be empty?

**Review by z-silver — CHANGES_REQUESTED** [review]

File: server/database/migrations/000003_map.sql (hunk 1 above)
Comment (EduardoLR10) [reply server/database/migrations/000003_map.sql:2]: You are bringing `Maybe not` to this conversation. For the sake of speed, I will say we use this for the 2D-array-using-1D-array arithmetic, so this needs to stay.

File: client/src/game/physics.zig [hunk 2]
```diff
@@ -43,13 +69,13 @@ pub const character = struct {
         if (velocity.y > 0) FrictionVector.y = -FrictionVector.y;
         if (velocity.z > 0) FrictionVector.z = -FrictionVector.z;
 
-        velocity.* = rm.vector3Add(velocity.*, FrictionVector);
+        velocity.* = velocity.*.add(FrictionVector);
```
Comment (z-silver) [inline client/src/game/physics.zig:72]:
```suggestion
        velocity.* = velocity.add(FrictionVector);
```

**Review by z-silver — CHANGES_REQUESTED** [review]

**Review by z-silver — APPROVED** [review]

**Pull request merged**